index.js

```diff
@@ -148,8 +149,22 @@ async function handleAlert(location, alertType) {
         fallbackMessage: `An alert to check on the bathroom at ${location.displayName} was not responded to. Please check on it`,
         fallbackToPhoneNumbers: location.fallbackNumbers,
         fallbackFromPhoneNumber: location.twilioNumber,
+        sirenParticleId: location.siren_particle_id,
       }
-      braveAlerter.startAlertSession(alertInfo)
+
+      if (location.siren_particle_id != null) {
```

Comment:
So Javascript is weird in that it has two different ways to check for equality. One which is the regular double-equals and the other which is the triple-equals. Generally speaking, what you normally want, is the triple-equals type, which means that checks like this would should actually be `!== null` and ones like on line 165 should be `=== null`, as funny as that looks.
Here's a good explanation, if you're interested in learning more: https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a